model/metrics/hardware/common.yaml

```diff
@@ -0,0 +1,61 @@
+groups:
+  # COMMON METRICS
+  - id: metric.hardware.attributes
+    type: attribute_group
+    brief: "Attributes for hardware metrics"
+    extends: hardware.attributes.common
+    attributes:
+      - ref: hw.type
+        requirement_level: required
+
+  - id: metric.hardware.energy
+    type: metric
+    metric_name: hw.energy
+    stability: experimental
+    brief: "Energy consumed by the component"
+    instrument: counter
+    unit: "J"
+    extends: metric.hardware.attributes
+
+  - id: metric.hardware.errors
+    type: metric
+    metric_name: hw.errors
+    stability: experimental
+    brief: "Number of errors encountered by the component"
+    instrument: counter
+    unit: "{error}"
+    extends: metric.hardware.attributes
+    attributes:
+      - ref: error.type
+        brief: "The type of error encountered by the component"
+        examples: ['uncorrected', 'zero_buffer_credit', 'crc', 'bad_sector']
+        requirement_level: recommended
```

Comment:
```suggestion
        requirement_level:
          conditionally_required: if and only if an error has occured
        note: >
          The `error.type` SHOULD match the error code reported by the component,
          the canonical name of exception that occurred, or another low-cardinality error identifier.
          Instrumentations SHOULD document the list of errors they report.  
```

Reply:
I will update it as it makes more sense but I need to note that originally it was a `recommended`